resources/idlharness.js

```diff
@@ -2240,16 +2240,19 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
             {
                 assert_equals(exception, null, "Unexpected exception when evaluating object");
                 assert_equals(typeof obj, expected_typeof, "wrong typeof object");
+                var fn;
                 if (!member["static"]) {
                     if (!this.is_global() && !member.isUnforgeable) {
                         assert_inherits(obj, member.name);
                     } else {
                         assert_own_property(obj, member.name);
                     }
+                    fn = obj[member.name];
                 }
                 else
                 {
-                    assert_false(member.name in obj);
+                    assert_own_property(obj.constructor, member.name, "constructor does not have property:");
```

Comment:
Ok.  Your suggestion is fine, I'll update the commit.  Also a bit unclear how to separate messages.  I added the trailing ":" because otherwise it just runs into the next text in the error chain.  Result I get with combo testharness and mocha I'm using would look like:
```
AssertionError: assert_own_property: Interface object must have static operation as own property: expected property "foo" missing: expected 1 to equal 0
    at ...
```